app/controllers/devise_recognisable/sessions_controller.rb

```diff
@@ -56,7 +56,8 @@ def check_for_authentication_token
       sign_in_ip: request.location.ip,
       sign_in_at: Time.now,
       user_agent: request.user_agent,
-      accept_header: request.headers["HTTP_ACCEPT"]
+      accept_header: request.headers["HTTP_ACCEPT"],
+      accept_language: request.headers["Accept-Language"].first
```

Comment:
`request.headers["Accept-Language"]` returns a string, e.g. "en-GB,en-US;q=0.9,en;q=0.8".
We want to store the whole string  so I don't think we want to call `.first`.